### PR TITLE
changed config to preference in the new version output

### PR DESCRIPTION
**Pull request by girishramnani**
## What is the purpose of this change? What does it change?

fixes the incorrect output when a new odo version is available
## Was the change discussed in an issue?
fixes https://github.com/openshift/odo/issues/1822

## How to test changes?
download an older version of release from odo's release and that should show the change

### CODE REVIEW

**Comment by amitkrout** [comment]
#1889 
/test integration

File: pkg/odo/cli/version/version.go [hunk 1]
```diff
@@ -132,6 +132,6 @@ func GetLatestReleaseInfo(info chan<- string) {
 			"to update manually, or visit https://github.com/openshift/odo/releases\n" +
 			"---\n" +
 			"If you wish to disable the update notifications, you can disable it by running\n" +
```
Comment (dharmit) [inline pkg/odo/cli/version/version.go]: Why not a simpler message like:
```
You can disable the update notifications by running 'odo preference set UpdateNotification false'
```
instead?

Comment (girishramnani) [reply pkg/odo/cli/version/version.go]: I actually prefer the original message 😅

**Pull request merged**